types/index.d.ts

```diff
@@ -55,6 +55,15 @@ export type StreamChatReactClient = Client.StreamChat<
   AnyType,
   StreamChatReactUserType
 >;
+export type StreamChatChannel = Client.Channel<
```

Comment:
since we're using the generic in the filterFunction type we no longer need this new type